src/components/dialogs/name-filter-or-contingency.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+import { elementExists } from '../../utils/rest-api';
+import CircularProgress from '@mui/material/CircularProgress';
+import CheckIcon from '@mui/icons-material/Check';
+import TextField from '@mui/material/TextField';
+import makeStyles from '@mui/styles/makeStyles';
+import PropTypes from 'prop-types';
+const useStyles = makeStyles((theme) => ({
+    inputLegend: {
+        backgroundImage:
+            'linear-gradient(rgba(255, 255, 255, 0.16), rgba(255, 255, 255, 0.16))',
```

Comment:
I guess this is copied from filter.js, but I don't understand this gradient where the two ends are the same color with the same alpha.